src/components/featured-products-section/featured-products-section.module.scss

```diff
@@ -0,0 +1,34 @@
+.root {
+    margin: 0 -2%;
```

Comment:
What's the `-2%`?

Reply:
we have `padding: 0 2%;` on `<main>`. If I remove it from `<main>` I need to set padding for all pages over the app and all sections in the home page. 
There are rare cases when content is stretches to 100% width. I did it using negative margin values.

Reply:
I added it to all pages, because I thought it's the same everywhere. I forgot about the home page. Now I think it's better to remove it and set padding for each page separately, for example with a common css class.